dnf5.spec

```diff
@@ -158,6 +158,7 @@ BuildRequires:  pkgconfig(libsolvext) >= %{libsolv_version}
 BuildRequires:  pkgconfig(rpm) >= 4.17.0
 BuildRequires:  pkgconfig(sqlite3) >= %{sqlite_version}
 BuildRequires:  toml11-static
+BuildRequires:  zlib-level
```

Comment:
s/zlib-level/zlib-devel/

Reply:
My fingers are too fat for the keyboard.